enterprise-beans/4.0/_index.md

```diff
@@ -1,19 +1,21 @@
 ---
-title: "Jakarta Enterprise Beans 4.0 (under development)"
-date: 2020-mm-dd
+title: "Jakarta Enterprise Beans 4.0 (milestone)"
+date: 2020-06-12
```

Comment:
I would put today's date.  This is a "go public" date.  So, whenever we end up merging, then this date controls when it becomes public.  We're making the data public as soon as projects are ready.  So, I would just suggest 06-05.

Reply:
I have changed this to today's date. This does not display on the website. 

Let me know if I need to change the date inside the document too. I can rebuild the documents with revisiondate as "June 05, 2020".